### ISSUE TITLE
Fail to handle step definition with a placeholder

### ISSUE THREAD

**Issue by title-earnin** [reqnroll/Reqnroll.Rider#14]
<img width="901" alt="image" src="https://github.com/reqnroll/Reqnroll.Rider/assets/169252966/a03c1e31-a73e-40ea-9082-151364fde0cf">

This is a project that's created executing `dotnet new reqnroll-project -t xunit`.
The feature could detect only step without any placeholder, but report a warning on steps with placeholders.

## Rider version
2024.1.2
# Platform
Apple M3 Max Sonoma 14.5

## reqnroll
2024.1.5

## repository
https://github.com/title-earnin/reqnroll

**Comment by ajeckmans** [comment]
See https://github.com/reqnroll/Reqnroll.Rider?tab=readme-ov-file#limitations
This extension only supports RegEx step matching for now.

**Comment by wojciechrak** [comment]
@ajeckmans are there any plans for adding cucumber expressions?

**Comment by Socolin** [comment]
Yes, I don't have much time and I have other priorities lately. If someone wants to work on this I can help.

**Comment by ajeckmans** [comment]
I have a bit too much work with that one other change I need done in ReqnRoll.
Also my last change in this plugin was definitely a rollercoaster ^^

**Comment by wojciechrak** [comment]
If you can point to similar implementation (maybe in Visual Studio plugin?) then I'll try getting my hands dirty. Never worked on IDE plugin, though, so no promises.

**Comment by dominik-niebuhr** [comment]
> If you can point to similar implementation (maybe in Visual Studio plugin?) then I'll try getting my hands dirty. Never worked on IDE plugin, though, so no promises.

have you got it working already?

**Comment by dominik-niebuhr** [comment]
I've tried my best at #19
